src/App.jsx

```diff
@@ -88,11 +88,11 @@ function App() {
     console.log("Photodiode:", config.USE_PHOTODIODE);
     // If on desktop
     if (config.USE_ELECTRON) {
-      const electron = window.require("electron");
-      const renderer = electron.ipcRenderer;
-      setRenderer(renderer);
+      const ipcRenderer = window.require("electron").ipcRenderer;
```

Comment:
could we be fancy and do `const { ipcRenderer} = window.require("electron")` ? (literally not sure if that works, but it seems like it might)

Reply:
yes it works